src/windows/win-crowdstrike-fix-bootloop-v2.ps1

```diff
@@ -171,6 +193,44 @@ function CleanUpRegtransmsAndTxrblfFiles
     }
 }
 
+function FixRegistryCorruptions
+{
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]$RegFile
+    )
+    Log-Info "Attempting to fix registry config file with chkreg.exe..."
+    Log-Info "Making a backup of the original registry config file..."
+    $timestamp = Get-Date -Format "yyyyMMddTHHmmssffffZ"
+    $backupFileName = "$RegFile.backup-$timestamp"
+    $copySucceeded = $false
+    try {
+        Copy-Item $RegFile -Destination $backupFileName -ErrorAction Stop
```

Comment:
We should probably backup the .LOG1 and .LOG2 files as well.

Reply:
I see. I just added the logic to back them up as well.